### PR TITLE
docs(utils/cookies): encourage `*.server` filename

### PR DESCRIPTION
**Pull request by thomasverleye**
Prevent cookies from being imported to the client by adding the server suffix.

### CODE REVIEW

File: docs/utils/cookies.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const userPrefs = createCookie("user-prefs", {
 
 Then, you can `import` the cookie and use it in your `loader` and/or `action`. The `loader` in this case just checks the value of the user preference so you can use it in your component for deciding whether to render the banner. When the button is clicked, the `<form>` calls the `action` on the server and reloads the page without the banner.
 
-**Note:** We recommend (for now) that you create all the cookies your app needs in `app/cookies.js` and `import` them into your route modules. This allows the Remix compiler to correctly prune these imports out of the browser build where they are not needed. We hope to eventually remove this caveat.
+**Note:** We recommend (for now) that you create all the cookies your app needs in `app/cookies.server.ts` and `import` them into your route modules. This allows the Remix compiler to correctly prune these imports out of the browser build where they are not needed. We hope to eventually remove this caveat.
```
Comment (machour) [inline docs/utils/cookies.md:31]:
```suggestion
**Note:** We recommend (for now) that you create all the cookies your app needs in `app/cookies.server.js` and `import` them into your route modules. This allows the Remix compiler to correctly prune these imports out of the browser build where they are not needed. We hope to eventually remove this caveat.
```

The .js file is mentioned above

**Review by machour — COMMENTED** [review]
Thank you for this contribution. 

The file is mentioned above, do we need to update its name there as well?

**Comment by thomasverleye** [comment]
@machour updated the docs some more:
- Changed the imports to assure the `.server` suffix is added in the other code examples;
- Changed back to `.js` extension as you suggested.

**Review by machour — APPROVED** [review]

**Comment by thomasverleye** [comment]
@machour this one is hanging two weeks now, any idea when you people merge these kinda pr's? 🙇🏻‍♂️

File: docs/utils/cookies.md [hunk 2]
```diff
@@ -133,14 +133,14 @@ Cookies that have one or more `secrets` will be stored and verified in a way tha
 
 Secrets may be rotated by adding new secrets to the front of the `secrets` array. Cookies that have been signed with old secrets will still be decoded successfully in `cookie.parse()`, and the newest secret (the first one in the array) will always be used to sign outgoing cookies created in `cookie.serialize()`.
 
-```ts filename=app/cookies.ts
+```ts filename=app/cookies.server.js
```
Comment (MichaelDeBoey) [inline docs/utils/cookies.md:136]: Let's keep using TS

```suggestion
```ts filename=app/cookies.server.ts
```

File: docs/utils/cookies.md [hunk 3]
```diff
@@ -18,7 +18,7 @@ Let's say you have a banner on your e-commerce site that prompts users to check
 
 First, create a cookie:
 
-```js filename=app/cookies.js
+```js filename=app/cookies.server.js
```
Comment (MichaelDeBoey) [inline docs/utils/cookies.md:21]:
```suggestion
```js filename=app/cookies.server.ts
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: docs/utils/cookies.md [hunk 4]
```diff
@@ -28,7 +28,7 @@ export const userPrefs = createCookie("user-prefs", {
 
 Then, you can `import` the cookie and use it in your `loader` and/or `action`. The `loader` in this case just checks the value of the user preference so you can use it in your component for deciding whether to render the banner. When the button is clicked, the `<form>` calls the `action` on the server and reloads the page without the banner.
 
-**Note:** We recommend (for now) that you create all the cookies your app needs in `app/cookies.js` and `import` them into your route modules. This allows the Remix compiler to correctly prune these imports out of the browser build where they are not needed. We hope to eventually remove this caveat.
+**Note:** We recommend (for now) that you create all the cookies your app needs in `app/cookies.server.js` and `import` them into your route modules. This allows the Remix compiler to correctly prune these imports out of the browser build where they are not needed. We hope to eventually remove this caveat.
```
Comment (MichaelDeBoey) [inline docs/utils/cookies.md:31]:
```suggestion
**Note:** We recommend (for now) that you create all the cookies your app needs in `app/cookies.server.ts` and `import` them into your route modules. This allows the Remix compiler to correctly prune these imports out of the browser build where they are not needed. We hope to eventually remove this caveat.
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

**Review by MichaelDeBoey — APPROVED** [review]

**Comment by thomasverleye** [comment]
Broke this one by rebasing :/ see #5997 for new PR

**Pull request closed without merging**